lib/bitcoin/interfaces/IBitcoinFileReader.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Reader for bitcoin files from bitcoin data directory
+ */
+export default interface IBitcoinFileReader {
```

Comment:
For my understanding, what is the motivation for creating an interface here? For testing? Or are you anticipating a different implementation? I was under the impression that the same implementation would still be used even if we use Azure Files?

Reply:
Using an interface is optional, mainly created it for future proofing. If someone wants to use a reader that doesn't read from local file system, they should be able to implement this interface and have it working.